benchmark/sql_regressions/run_regressions.sh

```diff
@@ -0,0 +1,211 @@
+" > releases_mean.csv
+    rm -f regressions_db
+    touch regressions_db
+    sqlite3 regressions_db < ../"$logictest"/regressions.sql
+    cp ../"$logictest"/import.sql .
+    sqlite3 regressions_db < import.sql
+    echo "Checking for test regressions..."
+
+    duration_query_output=`sqlite3 regressions_db 'select * from releases_nightly_duration_change'`
+    result_query_output=`sqlite3 regressions_db 'select * from releases_nightly_result_change'`
+
+    duration_regressions=`echo $duration_query_output | sed '/^\s*$/d' | wc -l | tr -d '[:space:]'`
+    result_regressions=`echo $result_query_output | sed '/^\s*$/d' | wc -l | tr -d '[:space:]'`
+
+    if [ "$duration_regressions" != 0 ]; then echo "Duration regression found, $duration_regressions != 0" && echo $duration_query_output && exit 1; else echo "No duration regressions found"; fi
+    if [ "$result_regressions" != 0 ]; then echo "Result regression found, $result_regressions != 0" && echo $result_query_output && exit 1; else echo "No result regressions found"; fi
+}
+
+function build_dolt_release() {
+    pushd ../../ && \
+    rm ./.ci_bin/dolt && \
+    curl -A CURL_USER_AGENT:-dolt-installer -fL "$DOLT_RELEASE_URL" > dolt.tar.gz && \
+    tar zxf dolt.tar.gz && \
+    install dolt-linux-amd64/bin/dolt ./.ci_bin/ && \
+    popd && \
+    echo "Finished installing dolt from release:" && \
+    dolt version
+}
+
+function build_dolt_checkout() {
+    pushd ../../go && \
+    rm -rf ../.ci_bin/ && mkdir ../.ci_bin/ && \
+    go get -mod=readonly ./... && \
+    go build -mod=readonly -o ../.ci_bin/dolt ./cmd/dolt/. && \
+    popd && \
+    echo "Finished installing dolt from checkout:" && \
+    dolt version
+}
+
+function import_one_releases() {
+    test_num="$1"
+    dolt table import -u releases_dolt_results ../"$logictest_main"/temp/parsed"$test_num".json
+    dolt add releases_dolt_results
+    dolt commit -m "update dolt sql performance results ($DOLT_VERSION) ($test_num)"
+}
+
+function import_releases() {
+    dolt checkout releases
+    seq 1 $TEST_N_TIMES | while read test_num; do
+        import_one_releases "$test_num"
+    done
+    dolt sql -r csv -q "\
+select version, test_file, line_num, avg(duration) as mean_duration, result from dolt_history_releases_dolt_results where version=\"${DOLT_VERSION}\" group by line_num;\
+" > releases_mean.csv
+    dolt table import -u releases_dolt_mean_results releases_mean.csv
+    dolt add releases_dolt_mean_results
+    dolt commit -m "update dolt sql performance mean results ($DOLT_VERSION)"
+    dolt push origin releases
+
+    dolt checkout regressions
+    dolt merge releases
+    dolt add .
+    dolt commit -m "merge releases"
+    dolt push origin regressions
+}
+
+build_dolt_checkout
+
+rm -rf dolt-sql-performance
+dolt clone Liquidata/dolt-sql-performance
+
+(cd "$logictest_main" && setup)
+
+if [[ "$FAIL_ON_EXISTING_VERSION" == true ]]; then
+  (cd dolt-sql-performance && check_version_exists)
+fi
```

Comment:
@zachmu now bails if the repo has data with for the `DOLT_VERSION` and it's configurable through the UI